Mobile-Rounds/Screens/Admin/Stations/Index.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<Page
+    x:Class="Mobile_Rounds.Screens.Admin.Stations.Index"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Mobile_Rounds.Screens.Admin.Stations"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:breadcrumb="using:GoedWare.Controls.Breadcrumb"
+    xmlns:vm="using:Mobile_Rounds.ViewModels.Admin.Stations"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+        <Grid.DataContext>
+            <vm:StationsViewModel />
+        </Grid.DataContext>
+
+        <!--Header-->
+        <breadcrumb:BreadcrumbControl
+            Grid.Row="1"
+            Grid.Column="1"
+            Grid.ColumnSpan="2"
+            Margin="10,0,0,0"
+            FontSize="30"
+            HomeCommand="{Binding GoHome}"
+            ItemCommand="{Binding CrumbCommand}" 
+            ItemsSource="{Binding Crumbs}">
+
+            <breadcrumb:BreadcrumbControl.SeperatorTemplate>
+                <DataTemplate>
+                    <SymbolIcon Symbol="Play"/>
+                </DataTemplate>
+            </breadcrumb:BreadcrumbControl.SeperatorTemplate>
+            <breadcrumb:BreadcrumbControl.HomeTemplate>
+                <DataTemplate>
+                    <TextBlock Text="HOME" FontSize="30" />
+                </DataTemplate>
+            </breadcrumb:BreadcrumbControl.HomeTemplate>
+            <breadcrumb:BreadcrumbControl.ItemTemplate>
+                <DataTemplate>
+                    <TextBlock Text="{Binding Title}" FontSize="30"/>
+                </DataTemplate>
+            </breadcrumb:BreadcrumbControl.ItemTemplate>
+        </breadcrumb:BreadcrumbControl>
+
+        <Grid HorizontalAlignment="Left" Height="680" Margin="712,217,0,0" VerticalAlignment="Top" Width="673"/>
```

Comment:
I know we will refactor the stuff, but I think we should use the "ColoredTile" control. Mostly because it keeps the theme consistent. It still provides the Command option, so there shouldn't be any issues there. Plus, these buttons were HUGE last time. I think too big for them even. But that is something we can get feedback on.